azure-pipelines/simple_test_framework.yml

```diff
@@ -30,33 +30,56 @@ jobs:
 
     steps:
       - bash: |
-          echo "##vso[task.prependpath]$CONDA/bin"
+              echo "##vso[task.setvariable variable=shellopts]errexit"
+        displayName: "Force exit on error (bash) "
+
+      - bash: |
+              echo "##vso[task.prependpath]$CONDA/bin"
         displayName: Add conda to PATH
 
       - bash: |
-          sudo chown -R $USER $CONDA
-          # conda update -y conda
+              sudo chown -R $USER $CONDA
```

Comment:
Split this step in two steps. The first one does the `chown` thing and must run exclusive for macOS. The second one does the update, but comment it out.